reservation/models.py

```diff
@@ -1,21 +1,28 @@
 from django.contrib.auth.models import User
 from django.db import models
-from smart_selects.db_fields import ChainedForeignKey, ChainedManyToManyField
+from django.utils import timezone
 
 from cinema.models import Schedule, Seat
 
 
+class ReservationManager(models.Manager):
+    def create_reservation(self, user, details, seat):
+        reservation = self.create(
+            user=user,
+            details=details,
+            seat=seat,
+            time_created=timezone.now()
+        )
```

Comment:
Uuuu, a manager! Nice!
Just out of curiosity what was the issue with the create method?
If it was done only for the time_created value this could also be achieved with a custom save method or with the auto_now_add attribute which I see you used below.
Would it help further, if it returned what self.create returns?

Reply:
Oh yeah, I wanted to use the return value futrther, but I forgot to return it from the manager